isos/bootstrap/bootstrap

```diff
@@ -59,21 +59,21 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
         cp /bin/tether ${MOUNTPOINT}/.tether/tether-bin
 
         # temporary hack for waiting the network
-        while ! ip addr show eno1 | grep " inet "; do sleep 1;done
+        while ! ip addr show eth1 | grep " inet "; do sleep 1;done
```

Comment:
looks like there are several places where "eth1" is used. It might be best to add a variable that references that string so that if we have to change that value again it will be a one line quick fix. Even though this is a hack until we can add our own nics.